fury/ui/elements.py

```diff
@@ -3052,3 +3055,231 @@ def directory_click_callback(self, i_ren, _obj, listboxitem):
                 self.set_slot_colors()
         i_ren.force_render()
         i_ren.event.abort()
+
+
+class Shape2D(UI):
+    """Create and Manage 2D Shapes.
+    """
+
+    def __init__(self, shape_type, position=(0, 0)):
+        """Init this UI element.
+
+        Parameters
+        ----------
+        shape_type : string
+            Type of shape to be created.
+        position : (float, float)
+            (x, y) in pixels.
+        """
+        self.shape_type = shape_type
```

Comment:
please, initialize `self.shape = None`. talking about that, `self.shape` should be renamed (e.g pencil, pen, tool, paint, brush, etc.... I let you pick)

Reply:
This property will hold a reference to an `Actor2D`, which is then added to the `DrawPanel`.
As I consider elements on `DrawPanel` as shapes, I have named it the same.